README.md

```diff
@@ -1 +1,87 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1) Starts the server with the test pages.
+2) Opens the specified browsers (locally or on SauceLabs).
+3) Runs mocha tests.
+4) Closes the browsers and stops the server after all tests are finished.
```

Comment:
https://github.com/georgiy-abbasov/functional-harness/blob/Implement-functional-harness/README.md#what-it-does
The list is written in the on line now:
![2015-12-16_1124](https://cloud.githubusercontent.com/assets/12047804/11835864/a85f10d4-a3e7-11e5-927b-f48413db3088.png)